From 6c2bc1a04cfe708615bcb9d77f69803bdf020324 Mon Sep 17 00:00:00 2001
From: Zsombor Gegesy <gzsombor@gmail.com>
Date: Sat, 17 Oct 2009 23:33:28 +0200
Subject: [PATCH 01/23] fix the problem in the downloadmanager, when the
 'delete' button is pressed, the downloaded item count is not updated

---
 src/downloadmanager.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git src/downloadmanager.cpp src/downloadmanager.cpp
index 84091f8..7be76fa 100644
--- src/downloadmanager.cpp
+++ src/downloadmanager.cpp
@@ -810,6 +810,7 @@ bool DownloadModel::removeRows(int row, int count, const QModelIndex &parent)
         }
     }
     m_downloadManager->m_autoSaver->changeOccurred();
+    m_downloadManager->updateItemCount();
     return true;
 }
 
-- 
2.44.0

